05/app.js

```diff
@@ -1,53 +1,67 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
+  paragraphs: {
+    p1: 0,
+  },
+  links: {
+    "/dolor.html": 0,
+  },
 };
 
 /* tutaj umieść swój kod */
 
+function counterClick(e) {
+  e.preventDefault();
+  if (e.target.tagName === "P") {
+    ++stats.paragraphs.p1;
+  } else {
+    ++stats.links["/dolor.html"];
+  }
```

Comment:
ok, ale to działa tylko dla `p1`, a tam jest więcej paragrafów czy linków.
Trzeba to dynamicznie pobierać np. 
```
if (e.target.tagName === "P") {
    const id = e.target.dataset.id
    ++stats.paragraphs[id];
  } else {
```

Dodatkowo musisz pamiętać, że nie wszystkie elementy w `stats` istnieją więc trzeba je utworzyć ;)